File: rewrite.py
import sys

with open("app/components/EtiquetasRapidas.js", "r", encoding="utf-8") as f:
    text = f.read()

idx = text.find("export default function EtiquetasRapidas() {")
if idx == -1:
    print("Could not find")
    sys.exit(1)

head = text[:idx]
with open("head.js", "w", encoding="utf-8") as f:
    f.write(head)

File: rewrite.js
const fs = require("fs");

const text = fs.readFileSync("app/components/EtiquetasRapidas.js", "utf-8");
const idx = text.indexOf("export default function EtiquetasRapidas() {");
if (idx === -1) process.exit(1);

const head = text.substring(0, idx);
fs.writeFileSync("head.js", head, "utf-8");
